### PR TITLE
[CloudPC] Add powerState in CloudPC entity

### PR DESCRIPTION
**Pull request by SleepIsImportant**

### CODE REVIEW

File: api-reference/beta/resources/cloudpc.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ Represents a cloud-managed virtual desktop. This Cloud PC is also enrolled into
 |onPremisesConnectionName|String|The Azure network connection that is applied during the provisioning of Cloud PCs.|
 |osVersion|[cloudPcOperatingSystem](../resources/cloudpcorganizationsettings.md#cloudpcoperatingsystem-values)|The version of the operating system (OS) to provision on Cloud PCs. Possible values are: `windows10`, `windows11`, and `unknownFutureValue`.|
 |partnerAgentInstallResults|[cloudPcPartnerAgentInstallResult](../resources/cloudpcpartneragentinstallresult.md) collection|The results of every partner agent's installation status on Cloud PC.|
+|powerState|[cloudPcPowerState](#cloudpcpowerstate-values)|The Power State of the Cloud PC. Possible values include running, poweredOff. This is only supported for Cloud PCs with the provisioningType set to 'shared'.|
```
Comment (AshleyYangSZ) [inline api-reference/beta/resources/cloudpc.md:65]: This is only supported for shift work CloudPCs.

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+                    "ApiChange": "Property",
+                    "ChangedApiName": "powerState",
+                    "ChangeType": "Addition",
+                    "Description": "Added the **powerState** property to the [cloudPc](https://learn.microsoft.com/en-us/graph/api/resources/cloudpc.md).",
+                    "Target": "cloudpc"
+                },
+                {
+                    "Id": "6680777e-7bec-43ff-9ad6-cb258a40783d",
+                    "ApiChange": "Enum type",
+                    "ChangedApiName": "cloudPCPowerState",
```
Comment (RuiHou105) [inline changelog/Microsoft.CloudManagedDesktop.json:16]:
```suggestion
                    "ChangedApiName": "cloudPcPowerState",
```

Comment (SleepIsImportant) [reply changelog/Microsoft.CloudManagedDesktop.json]: Fixed thaks

**Review by SleepIsImportant — COMMENTED** [review]
fixed

**Comment by Danielabom** [comment]
Contacted author to get info.

**Comment by Danielabom** [comment]
Contacted author to get info.

File: api-reference/beta/resources/cloudpc.md [hunk 3]
```diff
@@ -63,6 +63,7 @@ Represents a cloud-managed virtual desktop. This Cloud PC is also enrolled into
 |onPremisesConnectionName|String|The Azure network connection that is applied during the provisioning of Cloud PCs.|
 |osVersion|[cloudPcOperatingSystem](../resources/cloudpcorganizationsettings.md#cloudpcoperatingsystem-values)|The version of the operating system (OS) to provision on Cloud PCs. Possible values are: `windows10`, `windows11`, and `unknownFutureValue`.|
 |partnerAgentInstallResults|[cloudPcPartnerAgentInstallResult](../resources/cloudpcpartneragentinstallresult.md) collection|The results of every partner agent's installation status on Cloud PC.|
+|powerState|[cloudPcPowerStatus](#cloudpcpowerstatus-values)|The Power State of the Cloud PC. Possible values include `running`, `poweredOff`, `deallocated`, `hibernated`, `unknown`. This is only supported for shift work CloudPCs.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpc.md:66]:
```suggestion
|powerState|[cloudPcPowerStatus](#cloudpcpowerstatus-values)|The power state of a Cloud PC. The possible values are: `running`, `poweredOff`, `deallocated`, `hibernated`, `unknown`. This property only supports shift work CloudPCs.|
```

File: api-reference/beta/resources/cloudpc.md [hunk 4]
```diff
@@ -100,6 +101,16 @@ Represents a cloud-managed virtual desktop. This Cloud PC is also enrolled into
...
 |unknownFutureValue|Evolvable enumeration sentinel value. Do not use.|
 
+### cloudPcPowerStatus values
+
+|Member|Description|
+|:---|:---|
+|running|The Cloud PC status is running.|
+|poweredOff|The Cloud PC status is power off.|
+|deallocated|The Cloud PC status is deallocated.|
+|hibernated|The Cloud PCstatus is hibernated.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpc.md:111]: The members "deallocated" and "hibernated" are not mentioned in the CSDL or the api.md files. Please confirm if these are valid members.

**Review by Danielabom — CHANGES_REQUESTED** [review]
Thank you for working on this PR @SleepIsImportant. Please review my comments.

**Comment by SleepIsImportant** [comment]
> Thank you for working on this PR @SleepIsImportant. Please review my comments.

@Danielabom Seems i've resolved all comments, could you pls kindly help check it again

**Pull request merged**